### PR TITLE
To publish message key from Output plugin

### LINKED ISSUE #90: Output plugin not publishing message key
Output plugin is not publishing message key to kafka topic.

### PR DESCRIPTION
**Pull request by vk9**
fixes #90

### CODE REVIEW

File: lib/fluent/plugin/out_kafka.rb [hunk 1]
```diff
@@ -162,7 +162,7 @@ def emit(tag, es, chain)
         value = @formatter_proc.call(tag, time, record)
 
         log.on_trace { log.trace("message send to #{topic} with key: #{partition_key} and value: #{value}.") }
-        producer.produce(value, topic: topic, partition_key: partition_key)
+        producer.produce(value, topic: topic, key: partition_key)
```
Comment (repeatedly) [inline lib/fluent/plugin/out_kafka.rb]: partition_key is for partition, not message key.
Does this change, using partition_key for message key, realize your purpose?

Comment (vk9) [inline lib/fluent/plugin/out_kafka.rb]: Ruby-kafka uses message_key for partitioning if partition_key is not set explicitly. 
Can you check [Ruby-kafka Usage](https://github.com/zendesk/ruby-kafka#producing-messages-to-kafka)

**Comment by vk9** [comment]
@repeatedly Added a new option for message_key, alongside partition_key

**Comment by repeatedly** [comment]
thx!

**Comment by vk9** [comment]
Would you be able to give new release of updated master ?

**Comment by repeatedly** [comment]
already done

**Comment by vk9** [comment]
thanks

**Pull request merged**